contracts/stake/src/contract.rs

```diff
@@ -270,6 +276,62 @@ impl StakingTrait for Staking {
         env.events().publish(("unbond", "amount"), stake_amount);
     }
 
+    fn unbond_deprecated(env: Env, sender: Address, stake_amount: i128, stake_timestamp: u64) {
+        sender.require_auth();
+        env.storage()
```

Comment:
This one is the new one, not the deprecated.

Reply:
that was a quick draft, I think I might be able to optimize both. Actually needed this because of how we check for pending rewards